src/flow_solver/flow_solver_cases/dipole_wall_collision.h

```diff
@@ -0,0 +1,59 @@
+#ifndef __DIPOLE_WALL_COLLISION_H__
+#define __DIPOLE_WALL_COLLISION_H__
+
+#include "periodic_turbulence.h"
+#include "dg/dg_base.hpp"
+
+namespace PHiLiP {
+namespace FlowSolver {
+
+#if PHILIP_DIM==1
+using Triangulation = dealii::Triangulation<PHILIP_DIM>;
+#else
+using Triangulation = dealii::parallel::distributed::Triangulation<PHILIP_DIM>;
+#endif
+
+template <int dim, int nstate>
+class DipoleWallCollision : public PeriodicTurbulence<dim,nstate>
+{
+public:
+    /// Constructor.
+    DipoleWallCollision(const Parameters::AllParameters *const parameters_input,
+                        const bool is_oblique_=false);
+
+    /// Destructor
+    ~DipoleWallCollision() {};
```

Comment:
I would advocate for removing default destructors throughout the PR for consistency with the rest of the code.